diff --git a/configure b/configure
index 8c9413b14c..909d3ce20f 100755
--- a/configure
+++ b/configure
@@ -5187,13 +5187,13 @@ else
   { $as_echo "$as_me:$LINENO: checking version of $CC" >&5
 $as_echo_n "checking version of $CC... " >&6; }
   ac_prog_version=`$CC -v 2>&1 | sed -n 's/^.*version \([egcygnustpi-]*[0-9.]*\).*$/\1/p'`
-  case $ac_prog_version in
-    '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    3.4* | 4.[0-9]* )
-       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-    *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+  #case $ac_prog_version in
+    #'') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
+    #3.4* | 4.[0-9]* )
+       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no #;;
+    #*) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
 
-  esac
+  #esac
   { $as_echo "$as_me:$LINENO: result: $ac_prog_version" >&5
 $as_echo "$ac_prog_version" >&6; }
 fi
@@ -5250,13 +5250,13 @@ else
   { $as_echo "$as_me:$LINENO: checking version of $MAKE" >&5
 $as_echo_n "checking version of $MAKE... " >&6; }
   ac_prog_version=`$MAKE --version 2>&1 | sed -n 's/^.*GNU Make[^0-9]*\([0-9][0-9.]*\).*$/\1/p'`
-  case $ac_prog_version in
-    '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    3.79* | 3.[89]*)
-       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-    *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+  #case $ac_prog_version in
+    #'') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
+    #3.79* | 3.[89]*)
+       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no #;;
+    #*) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
 
-  esac
+  #esac
   { $as_echo "$as_me:$LINENO: result: $ac_prog_version" >&5
 $as_echo "$ac_prog_version" >&6; }
 fi
@@ -5440,13 +5440,13 @@ else
   { $as_echo "$as_me:$LINENO: checking version of $SED" >&5
 $as_echo_n "checking version of $SED... " >&6; }
   ac_prog_version=`$SED --version 2>&1 | sed -n 's/^.*GNU sed version \([0-9]*\.[0-9.]*\).*$/\1/p'`
-  case $ac_prog_version in
-    '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    3.0[2-9]*|3.[1-9]*|[4-9]*)
-       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-    *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+  #case $ac_prog_version in
+    #'') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
+    #3.0[2-9]*|3.[1-9]*|[4-9]*)
+       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no #;;
+    #*) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
 
-  esac
+  #esac
   { $as_echo "$as_me:$LINENO: result: $ac_prog_version" >&5
 $as_echo "$ac_prog_version" >&6; }
 fi
